NBitcoin.Altcoins/Mogwai.cs

```diff
@@ -0,0 +1,237 @@
+﻿using NBitcoin;
+using NBitcoin.Crypto;
+using NBitcoin.DataEncoders;
+using NBitcoin.Protocol;
+using NBitcoin.RPC;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NBitcoin.Altcoins
+{
+	// Reference: https://github.com/mogwaicoin/mogwai/blob/master/src/chainparams.cpp
+	public class Mogwai : NetworkSetBase
+	{
+		public static Mogwai Instance { get; } = new Mogwai();
+
+		public override string CryptoCode => "MOG";
+
+		private Mogwai()
+		{
+
+		}
+
+		public class MogwaiConsensusFactory : ConsensusFactory
```

Comment:
you do not need this.

Reply:
Thx, for the hint, I removed it in the latest commit, If you could accept merge request now, we would be happy. :-)